### PR TITLE
[IMP] Add Docs to helper objs & fix sphinx syntax

### PR DESCRIPTION
**Pull request by lasley**

### CODE REVIEW

**Comment by lasley** [comment]
Well crap the new Travis script doesn't fail when it's broken - https://travis-ci.org/LasLabs/python-cfssl/jobs/185644614#L276

How useless 😆

File: cfssl/models/config_client.py [hunk 1]
```diff
@@ -10,6 +10,18 @@ class ConfigClient(ConfigMixer):
...
+        """ Initialize a new Client Configuration.
+
+        Args:
+            sign_policy_default (:obj:`cfssl.PolicySign`): Default signing
+                policy for client to use.
+            sign_policies_add (:type:`iter` of :obj:`cfssl.PolicySign`):
+                Additional signing policies to use for the client.
+            auth_policies (:type:`iter` of :obj:`cfssl.PolicyAuth`): Auth
+                policies for the client.
+            remote (:type:`iter` of :obj:`cfssl.Host`): Remote hosts that
```
Comment (tedsalmon) [inline cfssl/models/config_client.py]: s/remote/remotes

**Review by tedsalmon — APPROVED** [review]
👍  LGTM! Nice docs :)

One minor spelling mistake ;)

**Comment by tedsalmon** [comment]
Awh crap. I loaded this before you switched it back to WIP. My bad!

**Comment by lasley** [comment]
Meh no worries, I'll just merge after making my few fixes for the build + your's. I noticed a bug but didn't update the tests.

**Comment by lasley** [comment]
Alright assuming this passes, I'll merge. Looking forward to seeing what this looks like in the docs.

**Pull request merged**